packages/gen-ai-hub/src/client/openai/openai-types.ts

```diff
@@ -444,7 +444,7 @@ export interface OpenAiChatCompletionChoice extends OpenAiCompletionChoice {
   /**
    * Completion chat message.
    */
-  message: OpenAiChatMessage;
+  message: OpenAiChatAssistantMessage;
```

Comment:
How did this type change occur?

Reply:
As discussed, this is avoid unnecessary check in the convenient method, the type will eventually be corrected in a followup PR